### PR TITLE
Relates to #140: Update readme to show use of Substrate Node Template

### PR DESCRIPTION
**Pull request by ltfschoen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,10 +46,10 @@ yarn start
 
 ### Terminal 3 - Node
 
-Follow up installation instructions from the [Polkadot repo](https://github.com/paritytech/polkadot)
+Follow up installation instructions from the [Substrate Node Template repo](https://github.com/substrate-developer-hub/substrate-node-template)
 
 ```sh
-polkadot --dev --telemetry-url ws://localhost:8000/submit
+./target/release/node-template --dev --telemetry-url 'ws://localhost:8000/submit 0'
```
Comment (dvdplm) [inline README.md:52]: `'ws://localhost:8000/submit 0' ` is a good change.

File: README.md [hunk 2]
```diff
@@ -46,10 +46,10 @@ yarn start
 
 ### Terminal 3 - Node
 
-Follow up installation instructions from the [Polkadot repo](https://github.com/paritytech/polkadot)
```
Comment (dvdplm) [inline README.md:49]: I think we want to stick with polkadot as the example; devs usually understand they can run any substrate chain.

**Review by dvdplm — CHANGES_REQUESTED** [review]
Please revert the switch to node template for the docs.

**Comment by jsdw** [comment]
Thanks for your PR! As it happens, I've probably covered these changes in the README from the `jsdw-sharding` branch, which we are planning to get merged to master in the coming days! I'd suggest closing this, and we can re-open a new issue or PR if the updated readme is unclear :)

**Pull request closed without merging**